src/Routes.tsx

```diff
@@ -0,0 +1,18 @@
+import * as React from 'react';
+import { HashRouter as Router, Route } from 'react-router-dom';
+
+import App from './App';
+import {Example} from './Examples';
+
+function Routes() {
+  return (
+    <Router>
+      <div>
+        <Route exact={true} path="/" component={App} />
+        <Route path="/example/:name" component={Example} />
```

Comment:
nit: RESTful routing convention is (usually) to pluralize resource names. What do you think of `"/examples/:name"` instead?

Reply:
Sure